### ISSUE TITLE
What is the convirt-mimic_resnet50.yml?

### ISSUE THREAD

**Issue by zoule41** [yangzhou12/BenchX#10]
The file BenchX-release\unifier\models\vlm_src\ConVIRT\builder.py in your code repository references CONFIG_FILEPATH = "config/templates/pretrain/convirt-mimic_resnet50.yml". 
Is this configuration file available for open-sourcing? 
Without it, reproducing the zero-shot classification and retrieval results would not be feasible.

**Comment by yangzhou12** [comment]
Given your interest, as well as that of others, we’re currently updating the codebase and will release the configuration files for image-text retrieval soon—please stay tuned. While we do not plan to further extend the code to support zero-shot classification, as it was not a focus of our paper, you should be able to adapt the image-text retrieval code to suit that purpose.

**Comment by yangzhou12** [comment]
We have updated the code and configuration files to support image-text retrieval. You should be able to reproduce the reported results by following the steps below (note that minor differences may occur due to variations in package versions and environment):

1. Download the pre-trained checkpoints from [## Pre-Trained Checkpoints](https://github.com/yangzhou12/BenchX/tree/release?tab=readme-ov-file#pre-trained-checkpoints).
2. Prepare the MIMIC-CXR dataset by selecting the relevant images and reports as specified in [mimic_5x200_ids.csv](datasets/MIMIC_5x200/mimic_5x200_ids.csv).
3. Run the following script:
`python bin/test_zeroshot.py configs/retrieval/<dataset_name>/<model_name>.yml ckpt_dir=<path_to_checkpoint>`

Please let me know if you have any problems.

**Comment by wangzw92** [comment]
> We have updated the code and configuration files to support image-text retrieval. You should be able to reproduce the reported results by following the steps below (note that minor differences may occur due to variations in package versions and environment):
> 
> 1. Download the pre-trained checkpoints from [## Pre-Trained Checkpoints](https://github.com/yangzhou12/BenchX/tree/release?tab=readme-ov-file#pre-trained-checkpoints).
> 2. Prepare the MIMIC-CXR dataset by selecting the relevant images and reports as specified in [mimic_5x200_ids.csv](datasets/MIMIC_5x200/mimic_5x200_ids.csv).
> 3. Run the following script:
>    `python bin/test_zeroshot.py configs/retrieval/<dataset_name>/<model_name>.yml ckpt_dir=<path_to_checkpoint>`
> 
> Please let me know if you have any problems.

Hello, I would like to consult with you about the issue of image and text retrieval tasks. Based on the mimic_5x200_ids.csv you provided, I am https://www.physionet.org/content/mimic-cxr/2.0.0/#files -Find the corresponding report content in the mimic-cxr-reports.zip provided by the panel. But some of them are findings that contain detailed descriptions, while others are impressions that only contain conclusions. May I ask if the report content in this CSV file is a mixture of two. Does it have an impact on performance evaluation.

**Comment by yangzhou12** [comment]
Thank you for your question and for looking into the dataset carefully.

Yes, the reports associated with the mimic_5x200_ids.csv file include the full report text, which may consist of findings, impressions, or both, depending on the original report content in MIMIC-CXR. We did not explicitly filter or separate reports based on whether they contained only findings, only impressions, or both.

We selected 200 image-report pairs by filtering out samples without any reports, ensuring label diversity using the CheXpert labels, and confirming they were not part of the pre-training data. However, we did not manually verify the structure of each report.

We think the variation in report content (e.g., detailed findings vs. concise impressions) could influence retrieval performance, but we did not conduct experiments specifically analyzing the impact of different report sections on performance.

**Comment by wangzw92** [comment]
> Thank you for your question and for looking into the dataset carefully.
> 
> Yes, the reports associated with the mimic_5x200_ids.csv file include the full report text, which may consist of findings, impressions, or both, depending on the original report content in MIMIC-CXR. We did not explicitly filter or separate reports based on whether they contained only findings, only impressions, or both.
> 
> We selected 200 image-report pairs by filtering out samples without any reports, ensuring label diversity using the CheXpert labels, and confirming they were not part of the pre-training data. However, we did not manually verify the structure of each report.
> 
> We think the variation in report content (e.g., detailed findings vs. concise impressions) could influence retrieval performance, but we did not conduct experiments specifically analyzing the impact of different report sections on performance.


Thank you for your answer. This will be of great help to my future work.
